src/aliceVision/robustEstimation/ACRansac.hpp

```diff
@@ -1,6 +1,8 @@
 // This file is part of the AliceVision project.
 // Copyright (c) 2016 AliceVision contributors.
-// Copyright (c) 2012 openMVG contributors.
```

Comment:
ACRansac belong to OpenMVG since 2012.
So please does not change the date to 2014, or consider to add back:
```// Copyright (c) 2012, 2016 Pierre MOULON.```

Reply:
Git history edited to fix it.